src/store/modules/job/actions.ts

```diff
@@ -51,13 +50,19 @@ const actions: ActionTree<JobState, RootState> = {
         "shopId_fld1_grp": "2",
         "shopId_fld1_op": "empty"
       } as any,
-      "fieldList": [ "systemJobEnumId", "runTime", "tempExprId", "parentJobId", "serviceName", "jobId", "jobName", "statusId", "cancelDateTime", "finishDateTime", "startDateTime" ],
+      "fieldList": [ "systemJobEnumId", "runTime", "tempExprId", "parentJobId", "serviceName", "jobId", "jobName", "statusId", "cancelDateTime", "finishDateTime", "startDateTime" , "enumTypeId" ],
       "noConditionFind": "Y",
       "viewSize": payload.viewSize,
       "viewIndex": payload.viewIndex,
       "orderBy": "runTime DESC"
     }
 
+    if(payload.statusId && (payload.statusId.length >= 3 || payload.statusId.length == 0)) {
```

Comment:
We can only check for `payload.statusId.length > 0`, and if yes then changing the `inputFields.statusId` value otherwise not.

Reply:
Sure, sir, will do.